src/services/auth/auth.controller.test.ts

```diff
@@ -140,37 +140,43 @@ describe('AuthController', () => {
 
   describe('/POST /request_hive_account', () => {
     it('creates a Hive account successfully', async () => {
+
+      const hiveUsername = 'test_user_id'
   
       // Make the request to the endpoint
       return request(app.getHttpServer())
         .post('/v1/auth/request_hive_account')
-        .send({ username: 'test_user_id' })
+        .send({ username: hiveUsername})
         .set('Authorization', 'Bearer <your_mocked_jwt_token>')
         .expect(201)
-        .then(response => {
+        .then(async response => {
           expect(response.body).toEqual({
             block_num: 1,
             expired: false,
             id: "id",
             trx_num: 10,
           });
+          
+          expect(await hiveService.isHiveAccountLinked('singleton/bob/did', hiveUsername)).toBe(true)
```

Comment:
The test function is already `async`. I think making a few variables and `await`ing the promises is better. I guess the same applies in many places actually. I'd recommend to do the refactor in a different PR.

Reply:
Maybe so; I see no harm in doing it this way for now. Maybe a task for after release

Reply:
I think this is fine for this PR, but next time you update a test like this, I think it is worth taking the extra time to clean it up a little.